packages/ui/components/ui/select-account.tsx

```diff
@@ -33,52 +33,43 @@ export const SelectAccount = ({ getAddrByIndex }: SelectAccountProps) => {
   }, [index, ephemeral]);
 
   return (
-    <>
-      {!address ? (
-        <></>
-      ) : (
-        <div className='flex w-full flex-col'>
-          <AccountSwitcher account={index} onChange={setIndex} />
+    address && (
+      <>
+        <AccountSwitcher account={index} onChange={setIndex} />
 
-          <div className='mt-4 flex items-center justify-between gap-1 break-all rounded-lg border bg-background px-3 py-4'>
-            <div className='flex items-center gap-[6px] overflow-hidden'>
-              <div className='shrink-0'>
-                <AddressIcon address={address} size={24} />
-              </div>
-
-              <p className='truncate text-sm'>
-                <AddressComponent address={address} ephemeral={ephemeral} />
-              </p>
-            </div>
-            <CopyToClipboardIconButton text={bech32Address(address)} />
+        <div className='mt-4 flex items-center justify-between gap-1 rounded-lg border bg-background px-3 py-4'>
+          <AddressIcon address={address} size={24} />
```

Comment:
The identicon looks to have lost some roundness here: 
<img width="121" alt="Screenshot 2024-03-28 at 10 31 08 AM" src="https://github.com/penumbra-zone/web/assets/16624263/95be2301-9afa-4b38-87fa-58a73d6b096f">

Reply:
this is a failure in identicon, have a fix that isn't PR'd yet